content/raw/chat/username_avatar.md

```diff
@@ -91,15 +91,16 @@ Prerequisites: User is in onboarding process or has already set up a profile and
 - Users MUST be able to set a profile photo for their profile
 - Profile photos MUST be visible to all users
 
-- A identicon ring MUST be composed of 10 color segments
+- Unless a profile has a registered ENS username, it MUST show the identicon ring around the profile avatar
```

Comment:
The ring should be shown even if the user has a ENS profile, ENS usernames are not immutable, for example:

1) ENS name blah.eth is registered with PK1
2) PK1 contacts user A, no ring is shown because `blah.eth` is registered to PK1, PK1 uses profile picture X
3) ENS name blah.eth is then registered to a different public key, PK2
4) PK2 contacts user A, using profile picture X

At this point you can't tell PK1 & PK2 apart, so they can be impersonated.

The ring should always be shown in my opinion, the only exception could possibly be verified contacts (not mutual), but in my opinion is not worth it, better to always be consistent so the user understands the purpose of it, but that's just my 2p, though for ENS we should def show the ring.